compare.py

```diff
@@ -63,33 +48,27 @@ def main():
     time_final_dev = time_final_dev.split('\t')[1]
     time_final_stable = time_final_stable.split('\t')[1]
 
-
     split_minutes_seconds_dev = re.split('[a-zA-Z]+', time_final_dev[:-1]) 
     split_minutes_seconds_stable = re.split('[a-zA-Z]+', time_final_stable[:-1]) 
 
-    time_stable_minutes = 0
-    time_dev_minutes = 0
-    minutes_multiplier = 1/60
-    
-
-
-    for i in range(len(split_minutes_seconds_dev) - 1, -1, -1):
-        time_dev_minutes += (minutes_multiplier * float(split_minutes_seconds_dev[i]))
-        minutes_multiplier *= 60
+    time_stable_sec = float(split_minutes_seconds_stable[1]) + (60 * float(split_minutes_seconds_stable[0]))
+    time_dev_sec = float(split_minutes_seconds_dev[1]) + (60 * float(split_minutes_seconds_stable[0]))
     
-    minutes_multiplier = 1/60
-    for i in range(len(split_minutes_seconds_stable) - 1, -1, -1):
-        time_stable_minutes += (minutes_multiplier * float(split_minutes_seconds_stable[i]))
-
-
     # Percent change on the latest branch wrt base branch
-    percent_change_time = f'{round(((time_dev_minutes - time_stable_minutes) / time_stable_minutes), 2) * 100}%'
-
+    percent_change_time = f'{round(((time_dev_sec - time_stable_sec) / time_stable_sec), 2) * 100}%'
 
     previous_data = json.load(previous_file)
     current_data = json.load(current_file)
     trigger_metadata_str = trigger_metadata_file.read()
 
+    summary_list.append("\n")
+    summary_list.append("Summary Report For: " + str(previous_data['repoName']))
```

Comment:
Use f-strings instead of appending strings in the code.

Reply:
Done for all places.